dataactcore/scripts/raw_sql/link_broken_subaward_grants.sql

```diff
@@ -34,7 +33,9 @@ aw_pafa AS
         AND EXISTS (
             SELECT 1
             FROM unlinked_subs
-            WHERE UPPER(TRANSLATE(unlinked_subs.award_id, '-', '')) = UPPER(TRANSLATE(pafa.fain, '-', ''))
+            WHERE pafa.record_type != 1
+                AND UPPER(TRANSLATE(unlinked_subs.award_id, '-', '')) = UPPER(TRANSLATE(pafa.fain, '-', ''))
+                AND UPPER(unlinked_subs.awarding_sub_tier_agency_c) IS NOT DISTINCT FROM UPPER(pafa.awarding_sub_tier_agency_c)
```

Comment:
Have you run any tests to see if `IS NOT DISTINCT FROM` is faster or slower than `COALESCE`?

This comment applies to all instances of `IS NOT DISTINCT FROM`

Reply:
Both are very similar in time with `IS NOT DISTINCT FROM` being just a little faster.
-- 6 rows, 509.5s IS NOT DISTINCT
-- 6 rows, 551.1s COALESCE